doc/src/conf.py

```diff
@@ -51,17 +49,25 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.insert(0, os.path.abspath('../..'))
-sys.path.insert(0, os.path.abspath('ext'))
+
+# there were extensions used in the past which were installed into the 'ext' directory.
+# Currently no extension is in use but it might be activated again some time in the future
+# sys.path.insert(0, os.path.abspath('ext'))
 
 import kafe2
 
 print('kafe2 version:', kafe2.__version__)
 
-# -- General configuration ------------------------------------------------
+# import the information which is already in the pyproject.toml
+import tomllib
+with open("../../pyproject.toml", "rb") as f:
+    toml = tomllib.load(f)
 
-# style sheet customizations
-def setup(app):
-    app.add_css_file("style.css")
+pyproject = toml["project"]
+# extract list of authors from pyproject.toml and replace the first name by its first letter
+authors = ", ".join([i['name'].split(' ')[0][0] + ". " + i['name'].split(' ')[1] for i in pyproject['authors']])
```

Comment:
You might want to use the last part as last name. E.g. if there are multiple first names
```suggestion
authors = ", ".join([i['name'].split(' ')[0][0] + ". " + i['name'].split(' ')[-1] for i in pyproject['authors']])
```

Reply:
Done. This line might still not work perfectly for every case but it is getting the job done for the current list of maintainers.